AGENT.md

```
@@ -0,0 +1,103 @@
+- Use spaces for indentation in TypeScript/JavaScript/JSON files (2 spaces)
+- Use tabs for other files (8 spaces width)
+- Always insert final newline
+- Trim trailing whitespace
+- Single quotes for strings
+- Always use semicolons
+- Use 1tbs (one true brace style)
+- Follow existing naming patterns (camelCase for variables/functions, PascalCase for types/interfaces)
+- Use markdown reference variables for links in documentation
+
+## Package Structure
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)